scylla-gdb.py

```diff
@@ -5775,17 +5775,16 @@ def dump_reads_from_semaphore(semaphore):
         total = permit_stats()
 
         for permit in intrusive_list(permit_list):
+            schema_name = "*.*"
             try:
-                schema = permit['_schema']['_p']
+                if int(permit['_schema']['_p']) != 0x0:
```

Comment:
why the neeed to convert to `int`, can't you just use `if permit['schema']['_p']`

Reply:
ack. updated the code.